public/components/common/search/search.tsx

```diff
@@ -221,15 +217,12 @@ export const Search = (props: any) => {
                   showOptionList={
                     showSavePanelOptionsList &&
                     searchBarConfigs[selectedSubTabId]?.showSavePanelOptionsList
-                    
                   }
                   curVisId={curVisId}
-                  spanValue={spanValue}
                   setSubType={setSubType}
-                  metricMeasure={metricMeasure}
-                  setMetricMeasure={setMetricMeasure}
-                  setMetricLabel={setMetricLabel}
-                  metricChecked={metricChecked}
+                  isSaveAsMetricEnabled={
+                    isEqual(curVisId, 'line') && tempQuery.match(PPL_SPAN_REGEX) !== null
+                  }
```

Comment:
Can you explain what is this doing/why this change was made here? Any issue linking the bug here?

Reply:
This comes from linting autofix.